conf: fix parallel install

Do not reference the upsmon.conf.sample twice, otherwise install, with
a high number of make jobs, may fail, like so:
    http://autobuild.buildroot.net/results/256/2567e13cd5bc702bc3a38a1d6fc8e34022cc7db5/build-end.log

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

---
Upstream status:
    https://github.com/networkupstools/nut/pull/147

diff -durB nut-2.7.2.orig/conf/Makefile.am nut-2.7.2/conf/Makefile.am
--- nut-2.7.2.orig/conf/Makefile.am	2014-02-14 09:56:53.000000000 +0100
+++ nut-2.7.2/conf/Makefile.am	2014-08-17 11:33:46.804439240 +0200
@@ -2,7 +2,7 @@
 
 INSTALL_0600 = $(INSTALL) -m 0600
 
-SECFILES = upsmon.conf.sample upsd.conf.sample upsd.users.sample
+SECFILES = upsd.conf.sample upsd.users.sample
 PUBFILES = nut.conf.sample ups.conf.sample
 CGIPUB = hosts.conf.sample upsset.conf.sample upsstats.html.sample	\
  upsstats-single.html.sample
